resources/core/charts/k8s-dashboard-proxy/templates/deployment.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "name" . }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: 1
+  template:
+    metadata:
+      name: {{ template "name" . }}
+      namespace: {{ .Release.Namespace }}
+      annotations:
+        sidecar.istio.io/inject: "true"
+      labels:
+        app: {{ template "name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      selector:
+        app: {{ template "name" . }}
+        release: {{ .Release.Name }}      
+      serviceAccount:  {{ template "name" . }}
+      {{ if .Values.global.isLocalEnv }}
+      hostNetwork: true
+      {{ end }}
+      containers:
+      - image: {{ .Values.keycloak.image }}
+        name: {{ template "name" . }}
+        {{ if not .Values.global.isLocalEnv }}
+        resources:
+          limits:
+            memory: 256Mi
+          requests:
+            memory: 128Mi
+        {{ end }}
+        imagePullPolicy: IfNotPresent
+        args:
+          - --discovery-url=https://dex.{{ .Values.global.domainName }}
+          - --skip-openid-provider-tls-verify=true
+          - --client-id=kubernetes-dashboard
+          - --client-secret=94e412ebd7c9440f6947972a1bb35698f007ba2d6401d65a8c2e8490a453d472
+          - --resources=uri=/*|methods=GET,POST,PUT,DELETE
```

Comment:
Would be nice to have this secret parametrized to allow overriding on different envs